docs/README.rst

```diff
@@ -9,10 +9,10 @@ This file provides a quick guide on how to compile the Scrapy documentation.
 
 Setup the environment
 ---------------------
+.. highlight:: bash
```

Comment:
:lipstick: This should probably beginning of the file, between `:orphan:` and the document title. Same for `docs/intro/install.rst`.